api/services/Reddit.js

```diff
@@ -50,9 +51,11 @@ var makeRequest = async function (refreshToken, requestType, url, data, rateLimi
     formData: data
   };
   let response = await request(options).catch(function (error) {
-    console.log('Reddit error: ' + requestType + ' request sent to ' + url + ' returned ' + error.statusCode +
-      ' - ' + error.statusMessage + '\nForm data sent: ' + JSON.stringify(data));
-    throw {statusCode: 502, error: 'Reddit responded with status code ' + error.statusCode};
+    if (!silentErrors) {
+      console.log('Reddit error: ' + requestType + ' request sent to ' + url + ' returned ' + error.statusCode);
+      console.log('Form data sent: ' + JSON.stringify(data));
+    }
+    throw {statusCode: error.statusCode, error: '(Reddit response)'};
```

Comment:
I think we should still throw the error when `silenceErrors` is on, so that it's clear that an error occured. We can handle the error further down.